frontend/src/Main.test.tsx

```diff
@@ -88,7 +88,12 @@ describe('Main', () => {
   it('Can show state transitions', async () => {
     await waitForPageToLoad();
 
-    const { getByLabelText, getByText, getByTestId } = renderResult;
+    const {
+      getByLabelText,
+      getByText,
+      getAllByText,
```

Comment:
nit: remove this line

Reply:
@jkoren can you make the change alex requested and then push a new update with the line removed and with it retested? Then I can merge. Thanks!